File: programs-basic-java/src/test/java/com/local/numbertypes/TwinPrimeNumberTest.java
package com.local.numbertypes;

import static com.local.numbertypes.NumberType.TWIN_PRIME_NUMBER;

import java.util.List;

import org.junit.jupiter.api.Test;

class TwinPrimeNumberTest extends AbstractNumberTest {

	@Test
	void testTwinPrimeNumbers() {
		testNumbersTypeByFile(TWIN_PRIME_NUMBER, "twinprimenumbers.txt");
	}

	@Test
	void testTwinPrimeNumbersByList() {
		// Pass the order as 5
		testNumbersTypeByList(TWIN_PRIME_NUMBER, List.of(1,3,5,7), 2);
	}
}

File: programs-basic-kotlin/src/test/kotlin/com/local/numbertypes/TwinPrimeNumberTest.kt
package com.local.numbertypes

import org.junit.jupiter.api.Test

class TwinPrimeNumberTest : AbstractNumberTest() {
	@Test
	fun testTwinPrimeNumbers() {
		testNumbersTypeByFile(com.local.numbertypes.NumberType.TWIN_PRIME_NUMBER, "twinprimenumbers.txt")
	}

	@Test
	fun testTwinPrimeNumbersByList() {
		// Pass the order as 5
		testNumbersTypeByList(com.local.numbertypes.NumberType.TWIN_PRIME_NUMBER, java.util.List.of<Int>(1, 3, 5, 7), 2)
	}
}